cli/command/container/utils.go

```diff
@@ -92,6 +94,7 @@ func waitExitOrRemoved(ctx context.Context, dockerCli *command.DockerCli, contai
 			}
 		}
 	}()
+	<-started
```

Comment:
Hmmm, I see your point, but I don't think it's that necessary, though it doesn't hurt anything either.

Even function returns before goroutine is scheduled, at least the `Event` call is already made, so we could monitor the container correctly.  This goroutine only influences outputting data to `statusChan`.